packages/react/src/global-alert/GlobalAlert.stories.tsx

```diff
@@ -6,6 +6,7 @@ const meta: Meta<typeof GlobalAlert> = {
 	title: 'content/GlobalAlert',
 	component: GlobalAlert,
 	args: {
+		onDismiss: undefined,
```

Comment:
Weirdly, without doing this, Storybook creates its own function for `onDismiss` meaning that none of the examples were correct, because they showed a Close button

Reply:
We can disable `actions` for individual stories, I believe. Or only allow them for some props. I'm not sure that's the best approach here, though, and this is probably the simplest but just in case you were worried about having this setting in the documentation.